apis/Google.Cloud.BigQuery.V2/docs/history.md

```diff
@@ -9,6 +9,7 @@ New features:
 - Support for time-based partitioning by a specific field
```

Comment:
And also support for time-based partition for load and query jobs, it was all done on the same PR if I remember correctly but they are listed as different features, rightly so I think.

Reply:
Ah yes - I'd got "in query options" but not load. Fixing.